test/ambiguous.jl

```diff
@@ -49,8 +50,6 @@ code_native(io, ambig, (Int, Int))
 # Test that ambiguous cases fail appropriately
 @test precompile(ambig, (UInt8, Int)) == false
 cfunction(ambig, Int, (UInt8, Int))  # test for a crash (doesn't throw an error)
-@test length(code_lowered(ambig, (UInt8, Int))) == 2
-@test length(code_typed(ambig, (UInt8, Int))) == 2
```

Comment:
I felt `code_*` were separate features that didn't need to be tested here. But they have the same length as the result of `methods`.